routes/index.js

```diff
@@ -274,4 +255,41 @@ function postStnMngr(req, res, next) {
       next();
     });
   }
+}
+
+function getBadgeIn(req, res) {
+  var data = {
+    messageType: req.messageType,
+    message: req.message
+  }
+
+  res.render('badgein.njk',data);
+}
+
+function postBadgeIn(req, res, next) {
+  if (!req.body) {
+    return res.sendStatus(400);
+  }
+
+  var BadgeNumber = req.body.badgeNumber;
+
+  dbAPI.validateUser(BadgeNumber).then(function (result) {
+    console.log('BN = ' + BadgeNumber);
+
+    // this checks if the badge number is not in the database
+    if (result == undefined) {
+
+     // go to the registration page
+      res.redirect('/registration/' + BadgeNumber);
+      
+    }
+    
+    else {
+      req.messageType = "success";
+      req.message = "You successfully Bagde In";
```

Comment:
There is a misspelling in this sentence. Other than that looks good to me.

Reply:
I guess to keep the badge in success page, I can move latter if we don't need it.

Reply:
Sure